src/Npgsql/NpgsqlCommand.cs

```diff
@@ -777,9 +777,8 @@ async Task Unprepare(bool async, CancellationToken cancellationToken = default)
 
         using (connector.StartUserAction(cancellationToken))
         {
-            var sendTask = SendClose(connector, async, cancellationToken);
-            if (sendTask.IsFaulted)
-                sendTask.GetAwaiter().GetResult();
+            // Just wait for SendClose to complete since each statement takes no more than 20 bytes
+            await SendClose(connector, async, cancellationToken);
```

Comment:
`.ConfigureAwait(false)` (but it's ok, I'll fix it all once more during a final rebase of https://github.com/npgsql/npgsql/pull/5225)